ui/src/store/resource.ts

```diff
@@ -149,10 +149,10 @@ export const ResourceStore = types
 
   .views((self) => ({
     get filteredResources() {
-      const { resources, kinds, catalogs, categories, search } = self;
+      const { resources, kinds, catalogs, categories, search, sortBy } = self;
       const { selectedTags } = categories;
 
-      const filtered: IResource[] = [];
+      let filtered: IResource[] = [];
```

Comment:
Can we rename this to something better like `filteredItems` or `filteredResources`?